commands/user.go

```diff
@@ -315,3 +329,23 @@ auth_service: {{.AuthService}}`
 
 	return nil
 }
+
+func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error {
+	page, _ := command.Flags().GetInt("page")
+	perPage, _ := command.Flags().GetInt("per-page")
+	showAll, _ := command.Flags().GetBool("all")
+
+	tpl := `{{.Id}}: {{.Username}} ({{.Email}})`
+	for users, _ := c.GetUsers(page, perPage, ""); users != nil && len(users) > 0; users, _ = c.GetUsers(page, perPage, "") {
```

Comment:
What's the reason we're using the blank identifier for these `GetUsers` errors?

Reply:
@njkevlani would you be open to help with @mkraft's question above? Thanks!